controllers/idler/idler_controller.go

```diff
@@ -25,6 +28,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
```

Comment:
same here: no need for a blank line (let's keep the imports grouped)

Reply:
done in [777b019](https://github.com/codeready-toolchain/member-operator/pull/357/commits/777b019b9dc2a2da36de0b0e2754b8012b10f7d8)